src/api/models/themes.js

```diff
@@ -132,12 +133,14 @@ const init = async () => {
                 );
             }
 
+            let hasIndex = false;
             let indexLocation = getThemeFile('default', 'index.ejs');
             if (themeConfig?.configuration?.files?.index) {
                 indexLocation = getThemeFile(
                     theme,
                     themeConfig.configuration.files.index,
                 );
+                hasIndex = true;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** Wouldn't it be more sure to test the existence of the file?‏